src/index.js

```diff
@@ -11,7 +11,9 @@ import uuidv4 from "uuid/v4";
 const uuid = localStorage.getItem("uuid") || uuidv4();
 localStorage.setItem("uuid", uuid);
 
-const popup = FlagPopup.fullscreen({ uuid, isExtensionPopup });
+const language = navigator.language || "en";
```

Comment:
I'm thinking of something, I, for instance use english as my default browser language, even though I'm brazillian, and by looking at google analytics, I believe this is the case of most of our users as well:

<img width="668" alt="image" src="https://user-images.githubusercontent.com/792201/31867924-299b8bac-b776-11e7-9bd8-d718c0038fb1.png">

<img width="300" alt="image" src="https://user-images.githubusercontent.com/792201/31867917-148775a0-b776-11e7-9e2f-a38ff3ed68f9.png">

(those visits are not mine, I block google analytics with Ghostery hahaha)

I thought that probably brazillian users would want to see the extension portuguese, even though the main system language is english (well, I would lol).

So, maybe, we could use `navigator.languages` instead of `navigator.language`, because the former returns an array of locales. What I thought was searching this array, and if there is `pt-*`, use it, else use the first one or `en` as default.

What do you think? Would this be an anti-pattern because we are tricking the expected behaviour based on user preferences?

I wanted something like this because I really want to focus this extension for brazillian audience and 2018 elections, and seeing something in english might put away users (maybe the user doesn't know how to proper configure their browser language)

Reply:
well, I agree with you, I use English as the default language in my browser and I think this should be very common.

> What do you think? Would this be an anti-pattern because we are tricking the expected behaviour based on user preferences?

To be honest, I think this solution fits in what we need, then in a next iteration we could add a menu in order to the user chooses the language by himself.

Reply:
cool! Yeah, we can always get feedbacks and discover if it was a bad decision later (:

Reply:
yeah, also that could be a new feature to be added, `Send suggestions` 😄